api/apps/api/test/proxy.vector-tiles.e2e-spec.ts

```diff
@@ -109,78 +118,98 @@ describe('ProxyVectorTilesModule (e2e)', () => {
           .expect(HttpStatus.OK);
       });
       describe('Filter by guid', () => {
-        test.skip('guid country level', async () => {
+        test('guid country level', async () => {
```

Comment:
very minor, but I suggest to try to stick to test statements that describe what should (or should not) happen. BDD/Gherkin style is my personal favourite because of clarity and of being widely known, but in any case, something less cryptic than the current statement would be welcome 😉 

same for the l1 and l2 tests below. the "guid level > 2 should not be allowed" one is clear enough, instead.

and the `.description()` call just above this comment could be improved too (e.g. even just "When filtering by guid")...